src/main/java/codeu/model/store/basic/HashtagStore.java

```diff
@@ -117,4 +117,16 @@ public Hashtag getHashtag(UUID id) {
     return null;
   }
 
+  public List<Hashtag> getHashtagsFromContent(String title) {
+    String[] hashtagList = title.split(" ");
+    List hashtagResults = new ArrayList();
+    for (String hash : hashtagList) {
+      if (hash.contains("#")) {
```

Comment:
When we found a hashtag I think we should be adding it to the in-memory list of hashtags as well as adding it to Set<Hashtag> in the message

Reply:
I think String has a function called startsWith

Reply:
this function just points out the hashtags I do not have time to have it add to the in-memory list.